.env.example

```diff
@@ -425,7 +425,8 @@ DIRECTORY_IDS_TO_LOG=
 # Read more about it in the README.md
 NEXT_PUBLIC_SINGLE_ORG_SLUG=
 
-AKISMET_API_KEY=
```

Comment:
We're confident it's just us using this and it's blocked behind the `IS_SELF_HOSTED` variable so we can safely remove this